main.py

```diff
@@ -1,7 +1,28 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Actor, Genre
 
 
 def main() -> QuerySet:
-    pass
+    Genre.objects.create(name="Western")
```

Comment:
To much repetition in the code below - read p2 adn p3 of checklist and adjust accordignly.